frontend/src/store/actions/collection/collection.js

```diff
@@ -35,12 +35,13 @@ const getCollectionsByUserIdFailure = (error) => ({
     target: error,
 });
 
-export const getCollectionsByUserId = (userId) => (dispatch) => axios.get("api/collection/user", { params: userId })
+// params can be { id: user_id } or { id: user_id, paper: id }
+export const getCollectionsByUserId = (params) => (dispatch) => axios.get("/api/collection/user", { params })
```

Comment:
Is this working well without { params: { id: 1 } } ?

Reply:
@kosumin Yeah, actually, Eslint automatically changed it.
Also as I mentioned on the explanation and the comment of that code, now you can give the format { id: user_id, paper: id } as an input to the action.

Reply:
Good! Thank you